File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/domain/SeatType.cs
using System;

namespace Registration
{
    [Serializable]
    public class SeatType
    {
        public Guid SeatTypeId { get; private set; }
        public string SeatTypeName { get; private set; }
        public decimal UnitPrice { get; private set; }

        public SeatType() { }
        public SeatType(Guid seatTypeId, string seatTypeName, decimal unitPrice)
        {
            SeatTypeId = seatTypeId;
            SeatTypeName = seatTypeName;
            UnitPrice = unitPrice;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/SeatType.java
package com.microsoft.conference.registration.domain;

import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
public class SeatType {
    private String seatTypeId;
    private String seatTypeName;
    private BigDecimal unitPrice;

    public SeatType() {
    }

    public SeatType(String seatTypeId, String seatTypeName, BigDecimal unitPrice) {
        this.seatTypeId = seatTypeId;
        this.seatTypeName = seatTypeName;
        this.unitPrice = unitPrice;
    }
}
